[BUG] Marlin/src/MarlinCore.cpp:1292:1: error: expected '}' at end of input

Attempting to build for the CR10-V2 with the latest bugfix-2.0.x.

Configs:
https://github.com/MarlinFirmware/Configurations/tree/bugfix-2.0.x/config/examples/Creality/CR-10%20V2

```
In file included from Marlin/src/inc/../HAL/./AVR/HAL.h:24:0,
                 from Marlin/src/inc/../HAL/HAL.h:30,
                 from Marlin/src/inc/MarlinConfig.h:31,
                 from Marlin/src/MarlinCore.h:24,
                 from Marlin/src/MarlinCore.cpp:31:
Marlin/src/MarlinCore.cpp: In function 'void setup()':
Marlin/src/inc/../HAL/./AVR/fastio.h:114:31: error: expected ';' before 'do'
 #define OUT_WRITE(IO,V)       do{ SET_OUTPUT(IO); WRITE(IO,V); }while(0)
                               ^
Marlin/src/MarlinCore.cpp:957:5: note: in expansion of macro 'OUT_WRITE'
     OUT_WRITE(SUICIDE_PIN, !SUICIDE_PIN_INVERTING);
     ^
Marlin/src/MarlinCore.cpp:1292:1: error: expected '}' at end of input
 }
```

--- a/Marlin/src/MarlinCore.cpp
+++ b/Marlin/src/MarlinCore.cpp
@@ -953,7 +953,7 @@ void setup() {
   #endif
 
   #if HAS_SUICIDE
-    SETUP_LOG("SUICIDE_PIN")
+    SETUP_LOG("SUICIDE_PIN");
     OUT_WRITE(SUICIDE_PIN, !SUICIDE_PIN_INVERTING);
   #endif
 

